cli/crates/cli/tests/unique.rs

```diff
@@ -155,3 +159,159 @@ fn unique() {
 
     assert!(first_author.is_none());
 }
+
+pub const ACCOUNT_CREATE_MUTATION: &str = include_str!("graphql/unique/multiple-field/account-create.graphql");
+pub const ACCOUNT_UPDATE_MUTATION: &str = include_str!("graphql/unique/multiple-field/account-update.graphql");
+pub const ACCOUNT_DELETE_MUTATION: &str = include_str!("graphql/unique/multiple-field/account-delete.graphql");
+pub const ACCOUNT_QUERY_ONE: &str = include_str!("graphql/unique/multiple-field/account-query-one.graphql");
+pub const ACCOUNT_QUERY_PAGINATED: &str = include_str!("graphql/unique/multiple-field/account-query-paginated.graphql");
+
+#[test]
+fn unique_with_multiple_fields() {
+    let mut env = Environment::init(4017);
```

Comment:
4017 is already in use by the default directive (we need a better system for this), 4021 should be free